### PR TITLE
Updating the private net guide to reflect light client changes after …

### LINKED ISSUE #2487: Review and update the private chain guide
## Description
Review and update if necessary https://github.com/raiden-network/light-client/blob/master/raiden-ts/docs-source/private-chain.md

## Acceptance criteria
- 

## Tasks
- [ ]

### PR DESCRIPTION
**Pull request by agatsoh**
**Thank you for submitting this pull request :)**

Fixes #2487 

**Short description**

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [ ] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

**Comment by agatsoh** [comment]
@palango I have made the required changes, you can have a look. We can add any tag that is subsequently introduced later.

File: raiden-ts/docs-source/private-chain.md [hunk 1]
```diff
@@ -1,68 +1,86 @@
...
 2. Locate the version of the smart contracts used in that specific version of the Light Client
 
 ### Locate the smart contract version
 
 You can easily find out the version of the smart contracts used in your version of the Light Client by checking the `raiden-contracts` submodule in the Light Client GitHub repo.
+
 1. Navigate to the [GitHub] project page
-2. Locate the `raiden-ts` subdirectory and look for the `raiden-contracts` submodule *(something like`raiden-contracts @ dfc0da6`)*
-3. Look at the commit message and check if it includes the `raiden-contracts` version. If not, follow the submodule link to the contracts repository and locate the `setup.py` file. You'll find the version of the Raiden Contracts by looking at the version variable *(`VERSION = '0.33.3'`)*
+2. Locate the `raiden-ts` subdirectory and look for the `raiden-contracts` submodule _(something like`raiden-contracts @ 770c8cd`)_
```
[the remark this answers is not in the document]
Comment (agatsoh) [reply raiden-ts/docs-source/private-chain.md:39]: Not this stuff the `Blame` stuff, which I have already removed and have given the user easier way to just check the tags. Which means once they know which version of raiden contracts they have to use they navigate through the tags and checkout raiden repo with the appropriate tag.

File: raiden-ts/docs-source/private-chain.md [hunk 2]
```diff
@@ -1,68 +1,86 @@
...
 This guide targets users that want to use the Light Client SDK or dApp with a custom network setup. This includes private chain deployments of the Raiden Network or unsupported networks (e.g., Kovan).
 
 To use a private network you first have to deploy the contracts by hand.
 Start by following this tutorial on how to deploy and run Raiden Network on a private chain:
-- [Raiden on Private Network Tutorial] 
 
-After finishing the guide above you should have a running Raiden node along with a private chain with the Raiden Network smart contracts deployed. Now you can move on to setting up the Light Client on your private network.
+- [Raiden on Private Network Tutorial]
+
+After finishing the guide above you should have a running Raiden node along with a private chain with the Raiden Network smart contracts deployed. But before that please go through **Setting up the Light Client on a private network** just below to figure the correct version of `raiden-contracts` and subsequently of `raiden` which will help us deploy raiden-contracts correctly.
```
Comment (palango) [inline raiden-ts/docs-source/private-chain.md:23]: "deploy raiden-contracts" -> "deploy the raiden contracts"

File: raiden-ts/docs-source/private-chain.md [hunk 3]
```diff
@@ -1,68 +1,86 @@
...
 
 ### Locate the smart contract version
 
 You can easily find out the version of the smart contracts used in your version of the Light Client by checking the `raiden-contracts` submodule in the Light Client GitHub repo.
+
 1. Navigate to the [GitHub] project page
-2. Locate the `raiden-ts` subdirectory and look for the `raiden-contracts` submodule *(something like`raiden-contracts @ dfc0da6`)*
-3. Look at the commit message and check if it includes the `raiden-contracts` version. If not, follow the submodule link to the contracts repository and locate the `setup.py` file. You'll find the version of the Raiden Contracts by looking at the version variable *(`VERSION = '0.33.3'`)*
+2. Locate the `raiden-ts` subdirectory and look for the `raiden-contracts` submodule _(something like`raiden-contracts @ 770c8cd`)_
+   ![raiden-contracts](https://user-images.githubusercontent.com/15123108/127455881-16d61132-1ded-4299-99ee-6a87298408be.png)
```
Comment (palango) [inline raiden-ts/docs-source/private-chain.md:40]: I don't like having screenshots in docs like this, they are hard to update and (in my opinion) harder to make sense of than well written text.

@taleldayekh opinion?

Comment (taleldayekh) [reply raiden-ts/docs-source/private-chain.md:40]: I agree, whenever we've used screenshots they've quickly gotten outdated. Better to just use text.

Comment (andrevmatos) [reply raiden-ts/docs-source/private-chain.md:40]: IMO screenshots can be useful, but not like this; in the perfect world, we could gather screenshots directly from cypress run and update them from CI; maybe this is a nice-to-have feature for the future.

Comment (agatsoh) [reply raiden-ts/docs-source/private-chain.md:40]: Ok I will remove this image from here, as we a majority for that.

File: raiden-ts/docs-source/private-chain.md (hunk 2 above)
Comment (agatsoh) [reply raiden-ts/docs-source/private-chain.md:23]: done

File: raiden-ts/docs-source/private-chain.md (hunk 3 above)
Comment (agatsoh) [reply raiden-ts/docs-source/private-chain.md]: @palango I have removed all the images from  the guide which i had put.

**Comment by agatsoh** [comment]
@palango I have made the required changes please check it out.

File: raiden-ts/docs-source/private-chain.md [hunk 4]
```diff
@@ -72,83 +88,101 @@ import privateServicesDeployment from '@/deployment/deployment_services_private_
...
+
+Comment out `const contracts = await ConfigProvider.contracts();` and add `contractsInfo` just below the `storageOpts` variable.
+
+Finally with these changes you need to run
+
+```bash
+yarn workspace raiden-dapp serve
+```
+
+You might face issues with linting for that you will need to go in the `raiden-dapp` subfolder and run the following:
```
Comment (palango) [inline raiden-ts/docs-source/private-chain.md:167]: Why might this happen?

**Review by palango — COMMENTED** [review]
Looks good, thanks for the effort. 

Could you please squash all commits into one? The changes aren't so big as to warrant 6 commits.

File: raiden-ts/docs-source/private-chain.md (hunk 4 above)
Comment (agatsoh) [reply raiden-ts/docs-source/private-chain.md:167]: i have told the user to run the command 
`yarn workspace raiden-dapp serve`
In this what happens on few occassions is that we might get lint errors but the `serve` command considers it as actual errors and the dapp is not served. So if we just fix linting and then serve then it works. Dont know why this peculiar behavior but it does happen on and off.

**Review by palango — APPROVED** [review]
please squash the commit.

**Pull request merged**